build-system/cpp-cmake-conan/src/build.py

```diff
@@ -30,7 +30,13 @@ def safe_get_workspace_dir() -> str:
     try:
         return get_workspace_dir()
     except Exception:
-        return "."
+        return os.path.abspath(".")
+
+
+def get_build_folder(build_arch: str, host_arch: str):
```

Comment:
since this is used in both files, can we move it to a common location?

Reply:
Question is: Where? velocitas_lib or should we keep it in build tooling package and remove the copying of the py scripts from the root folder plus calling velocitas exec from the shell scripts?

Reply:
could be one additional module next to build and install_deps